packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import { Icon } from '../../1_atoms/Icon/Icon';
+import styles from './WalletContainer.module.css';
+
+type WalletContainerProps = {
+  name: string;
+  icon?: ReactNode;
+  tooltip?: string;
+  className?: string;
+  dataLayoutId?: string;
+};
+
+export const WalletContainer: FC<WalletContainerProps> = ({
+  name,
+  icon,
+  tooltip,
+  className,
+  dataLayoutId,
+}) => {
+  return (
+    <div
+      data-layout-id={dataLayoutId}
+      className={classNames(className, styles.walletContainer)}
+    >
+      <div className={styles.label}>
+        {name}
+        {tooltip && <Icon icon="info" size={10} />}
+      </div>{' '}
+      {icon && <div className={styles.icon}>{icon}</div>}
+    </div>
+  );
+};
```

Comment:
Change it to button and pass default button props too:

```suggestion
type WalletContainerProps = {
  name: string;
  icon?: ReactNode;
  tooltip?: string;
  dataLayoutId?: string;
} & React.ButtonHTMLAttributes<React.HTMLButtonElement>;

export const WalletContainer: FC<WalletContainerProps> = ({
  name,
  icon,
  tooltip,
  dataLayoutId,
  ...buttonProps
}) => {
  return (
    <button
      data-layout-id={dataLayoutId}
      className={classNames(className, styles.walletContainer)}
      {...buttonProps}
    >
      <div className={styles.label}>
        {name}
        {tooltip && <Icon icon="info" size={10} />}
      </div>{' '}
      {icon && <div className={styles.icon}>{icon}</div>}
    </button>
  );
};
```

Reply:
ah ok, i thought that we would wrap this container in a button outside where we plan to use it